include/laser_filters/range_filter.h

```diff
@@ -52,9 +52,26 @@ class LaserScanRangeFilter : public filters::FilterBase<sensor_msgs::LaserScan>
 
   double lower_threshold_ ;
   double upper_threshold_ ;
+  bool use_message_range_ ;
```

Comment:
Could you rename this variable (and the ROS param name) to `use_message_range_limits`? That should make it clearer what it does.